runtime-light/stdlib/crypto/crypto-functions.cpp

```diff
@@ -343,3 +347,97 @@ task_t<Optional<string>> f$openssl_decrypt(string data, const string &method, co
   }
   co_return string{response.inner.value.data(), static_cast<string::size_type>(response.inner.value.size())};
 }
+
+namespace {
+constexpr std::pair<const char *, tl::HashAlgorithm> HASH_ALGOS[] = {{"md5", tl::HashAlgorithm::MD5},       {"sha1", tl::HashAlgorithm::SHA1},
```

Comment:
Why not use `std::array` or `std::span`?

Reply:
fix